automation-swiftui/AutomationAssessment/ContentView.swift

```diff
@@ -148,30 +141,3 @@ struct ContentView: View {
 #Preview {
     ContentView()
 }
-
-enum Tag: Hashable, CaseIterable {
```

Comment:
I've moved this to its own file for readability- its a chunky enum that seems out of place here